### PR TITLE
feat: move paper wallet only to admin

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Move paper wallet-only printing to the admin, support the rest of coins](https://trello.com/c/9Li42AQV/934-move-paper-wallet-only-printing-to-the-admin-support-the-rest-of-coins)

File: new-lamassu-admin/src/pages/Blacklist/Blacklist.js [hunk 1]
```diff
@@ -66,6 +67,50 @@ const ADD_ROW = gql`
   }
 `
 
+const PaperWalletDialog = ({ onConfirmed, onDissmised, open, props }) => {
+  const classes = useStyles()
+
+  return (
+    <>
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Blacklist/Blacklist.js]: You can remove the `<>(...)</>` node

File: new-lamassu-admin/src/pages/Blacklist/Blacklist.js [hunk 2]
```diff
@@ -108,6 +154,8 @@ const Blacklist = () => {
 
   const rejectAddressReuse = complianceConfig?.rejectAddressReuse ?? false
 
+  const enablePaperWalletOnly = complianceConfig?.enablePaperWalletOnly ?? false
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Blacklist/Blacklist.js]: You can change line 155 as well, to simplify code

```suggestion
  const enablePaperWalletOnly = !!complianceConfig.enablePaperWalletOnly
```

File: new-lamassu-admin/src/pages/Blacklist/Blacklist.js [hunk 3]
```diff
@@ -172,6 +234,35 @@ const Blacklist = () => {
...
+                checked={enablePaperWalletOnly}
+                onChange={event => {
+                  if (enablePaperWalletOnly) {
+                    addressReuseSave({
+                      enablePaperWalletOnly: event.target.checked
+                    })
+                  } else {
+                    setConfirmDialog(true)
+                  }
+                }}
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Blacklist/Blacklist.js]:
```suggestion
                onChange={e => enablePaperWalletOnly
                  ? addressReuseSave({ enablePaperWalletOnly:  e.target.checked })
                  : setConfirmDialog(true)
                }
```

**Review by chaotixkilla — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Blacklist/Blacklist.js [hunk 4]
```diff
@@ -172,6 +232,33 @@ const Blacklist = () => {
...
+                      })
+                    : setConfirmDialog(true)
+                }
+                value={enablePaperWalletOnly}
+              />
+              <Label2>{enablePaperWalletOnly ? 'On' : 'Off'}</Label2>
+              <Tooltip width={304}>
+                <P>
+                  The "Enable paper wallet (only)" option means that all only
+                  paper wallets paper wallets will be printed for users, and
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Blacklist/Blacklist.js]: `paper wallets paper wallets` on repeat here. And `... means that all only ...` doesn't sound well

File: new-lamassu-admin/src/pages/Blacklist/Blacklist.js [hunk 5]
```diff
@@ -108,6 +152,8 @@ const Blacklist = () => {
 
   const rejectAddressReuse = complianceConfig?.rejectAddressReuse ?? false
 
+  const enablePaperWalletOnly = !!complianceConfig?.enablePaperWalletOnly
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Blacklist/Blacklist.js:155]: You can change the variable on line 153 to match the `!!` format of this new one. For consistency's sake

**Review by chaotixkilla — CHANGES_REQUESTED** [review]

**Review by chaotixkilla — APPROVED** [review]

**Review by chaotixkilla — APPROVED** [review]

**Pull request merged**